src/Microsoft.AspNet.Mvc.Xml/Formatters/XmlDataContractSerializerInputFormatter.cs

```diff
@@ -106,7 +126,7 @@ protected virtual XmlReader CreateXmlReader([NotNull] Stream readStream)
         /// <returns>The <see cref="XmlObjectSerializer"/> used during deserialization.</returns>
         protected virtual XmlObjectSerializer CreateDataContractSerializer(Type type)
```

Comment:
Should CreateDataContractSerializer be renamed to CreateSerializer for symmetry with the output formatter? Should it return DataContractSerializer instead of XmlObjectSerializer?  The parameter also needs a NotNull attribute.  Here is what it might look like:
```csharp
protected virtual DataContractSerializer CreateSerializer([NotNull] Type type)
```